app/config/parameters.yml.dist

```diff
@@ -8,12 +8,21 @@ parameters:
     database_path:     "%kernel.root_dir%/app.sqlite"
 
     phpcr_backend:
-        # use Jackalope Doctrine DBAL
+        # if you're using Jackalope Doctrine DBAL
         type: doctrinedbal
         connection: default
         caches:
             meta: doctrine_cache.providers.phpcr_meta
             nodes: doctrine_cache.providers.phpcr_nodes
+        
+        # if you're using Jackalope Jackrabbit, comment the lines before
+        # and uncomment the lines after this comment
+        #logging: true
+        #profiling: true
+        #type: jackrabbit
+        #url: http://localhost:8080/server/
+        #parameters:
+        #    jackalope.check_login_on_server: false
```

Comment:
actually thought about this once more. in dev it makes sense to have this set to true to ease debugging and in production you want it off for performance which is exactly what we are doing by default:
https://github.com/doctrine/DoctrinePHPCRBundle/blob/master/DependencyInjection/DoctrinePHPCRExtension.php#L202

now right now with Doctrine DBAL this can cause issues if the DB has not yet been initialized. however as of Jackalope Doctrine DBAL 1.2 it should no longer be a problem if the "check" is enabled in dev mode and the DB is not yet initialized (to be confirmed).

so given that I would say lets remove the setting.